common/math.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "common.h"
+
+float approxSin(float x);
+
+inline float clamp(float x, float min, float max) {
+    return x<min ? min : x>max ? max : x;
+}
+
+inline float nxfabs(float x) {
+    return x < 0 ? -x : x;
```

Comment:
What's the point of replacing fabs with this function? GCC is able to see fabs calls and replace it with actual AArch64 VFP instructions.

Reply:
I was having troubles with including math.h in the nro builds when I was trying to use sin previously, so I assumed fabs would be the same. It seems to work fine so I've changed it now. In all likelihood I was just doing something wrong before.